File: java/composite/Employee.java
package composite;

public class Employee implements OrganisationUnit {

    private String name;
    private double salary;

    public Employee(String name, double salary) {
        this.name = name;
        this.salary = salary;
    }

    public double calculateTotalSalary() {
        return salary;
    }
}

File: C#/Composite/Employee.cs
namespace Composite;

public sealed class Employee : IOrganisationUnit
{
    private readonly string _name;
    private readonly double _salary;

    public Employee(string name, double salary)
    {
        _name = name;
        _salary = salary;
    }

    public double CalculateTotalSalary() => _salary;
}
